From 9e4c1ef73b4301c5f9dc918982dd4f1ffb3e1052 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Wed, 29 Jul 2026 15:40:57 -0700
Subject: [PATCH 08376/15333] apparmor: compressed_data not described in
 aa_get_data_from_compressed

Warning: security/apparmor/apparmorfs.c:501 function parameter 'compressed_data' not described in 'aa_get_data_from_compressed'

Warning: security/apparmor/apparmorfs.c:501 function parameter 'compressed_data' not described in 'aa_get_data_from_compressed'
Warning: security/apparmor/apparmorfs.c:501 function parameter 'compressed_data' not described in 'aa_get_data_from_compressed'

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202607240144.4iqulDF1-lkp@intel.com/
Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 security/apparmor/apparmorfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/apparmorfs.c b/security/apparmor/apparmorfs.c
index 152c7967ff1b..79b91c680e46 100644
--- a/security/apparmor/apparmorfs.c
+++ b/security/apparmor/apparmorfs.c
@@ -525,7 +525,7 @@ static int decompress_zstd(char *src, size_t slen, char *dst, size_t dlen)
  * @userbuf: user buffer to copy data from  (NOT NULL)
  * @buffer_size: size of user buffer
  * @pos: position write is at in the file (NOT NULL)
- * @compressed_data Ptr on compressed data. *compressed_data is allocated there
+ * @compressed_data: Ptr on compressed data. *compressed_data is allocated there
  *
  * Returns: kernel buffer containing copy of user buffer data or an
  *          ERR_PTR on failure.
-- 
2.55.0

